ledger/participant-integration-api/src/main/scala/platform/store/dao/events/EventsTableFlatEventsRangeQueries.scala

```diff
@@ -83,6 +44,26 @@ private[events] sealed abstract class EventsTableFlatEventsRangeQueries[Offset]
 
 private[events] object EventsTableFlatEventsRangeQueries {
 
+  private[events] def filterParams(
```

Comment:
real nice change! do we have unit tests to guard the this change in logic? (lot's of if-s always make me nervous :fearful: )

Reply:
there was ;)